platform/darwin/src/MGLFeature.mm

```diff
@@ -177,6 +177,18 @@ - (NSDictionary *)geoJSONDictionary {
     return mbgl::Feature{geometry};
 }
 
+- (mbgl::FeatureCollection)mbglFeatureCollection {
+    mbgl::FeatureCollection featureCollection;
+    featureCollection.reserve(self.shapes.count);
+    for (id <MGLFeaturePrivate> feature in self.shapes) {
+        if ([feature isMemberOfClass:[MGLShapeCollectionFeature class]]) {
```

Comment:
`-isKindOfClass:` is more forward-compatible than `-isMemberOfClass:`. Use the latter only if you want to avoid subclasses, for instance if you want to check for a shape collection that isn’t a shape collection feature.